subsys/mgmt/ec_host_cmd/backends/Kconfig

```diff
@@ -75,6 +75,14 @@ config EC_HOST_CMD_BACKEND_SHI_NPCX_ENHANCED_BUF_MODE
 	  single-byte output buffer can be selected/switched to generate a
 	  response to simultaneous Read/Write transactions.
 
+config EC_HOST_CMD_BACKEND_SHI_NPCX_CS_DETECT_WORKAROUND
+	bool
+	default y
+	#default y if SOC_NPCX9M7FB
```

Comment:
Does it really apply to all chips? `#default y if SOC_NPCX9M7FB` is this needed?

Reply:
It only needs to apply on NPCX9M7FB. Have updated the Kconfig for this configuration.